dashboard/test/ui/runner.rb

```diff
@@ -708,6 +713,8 @@ def cucumber_arguments_for_browser(browser, options)
   arguments += skip_tag('@webpurify') unless CDO.webpurify_key
   arguments += skip_tag('@pegasus_db_access') unless options.pegasus_db_access
   arguments += skip_tag('@dashboard_db_access') unless options.dashboard_db_access
+  arguments += skip_tag('@properties_encryption_key') if CDO.properties_encryption_key.blank?
```

Comment:
In lieu of giving this tag a name that embeds some kind of discouragement (which I have been unable to come up with), I would love to see a comment somewhere explaining that this tag is intended for backwards compatibility only and that new tests should be written against unencrypted allthethings levels if at all possible.

I'm not sure where the best place would be for such a comment, but this line seems like a good candidate.